### PR TITLE
Add Spanish translation for Intro -> 05 principles

### PR DESCRIPTION
**Pull request by dicortazar**

### CODE REVIEW

File: introduction/es/05-principles.asciidoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Estos principios son:
+* *Disponibilidad*
+* *Transparencia*
+* *Mentoría priorizada*
+* *Contribuciones voluntarias de código fuente*
+
+Echemos un vistazo a cada uno de estos principios en más detalle.
+
+
+=== Disponibilidad
```
Comment (dicortazar) [inline introduction/es/05-principles.asciidoc:17]: I'm not 100% sure about using this term. An alternative is "Apertura"... Ideas?

Comment (fioddor) [reply introduction/es/05-principles.asciidoc:17]: "Acceso" or "Acceso abierto".

- "Disponibilidad" blurs into "Availability"
- "Accesibilidad" blurs into "Accesibility", as for physically handicaped / diverse functional people.

Comment (dicortazar) [reply introduction/es/05-principles.asciidoc:17]: Sounds good, I'll go for "Acceso abierto".

**Comment by rrrutledge** [comment]
Hi @fioddor, how does this article look to you?

**Comment by fioddor** [comment]
I guess you mean the translation, @rrrutledge. It's ok.

**Comment by rrrutledge** [comment]
> I guess you mean the translation, @rrrutledge. It's ok.

Yes, the translation.  We're so glad to have you here, reviewing.  Previously I was trying my best with my limited Spanish.  Do you think it's OK enough to put on the website?  If so, then "Approve" this pull request?  Thank you!

**Comment by fioddor** [comment]
Unless I'm missing something, I guess I cannot approve because I don't have write access.

.
.
.
BTW, I understand your effort as the same happens to me in [FreedomBox wiki](https://wiki.debian.org/de/FreedomBox) with german  ;-)

**Comment by rrrutledge** [comment]
> I cannot approve because I don't have write access

Just sent an invite.  Does that work?  You've been giving good review.

**Review by fioddor — APPROVED** [review]

**Review by rrrutledge — APPROVED** [review]
Nice work!

**Pull request merged**